lib/util/emoji.spec.ts

```diff
@@ -1,5 +1,16 @@
+import { fromCodepointToUnicode, fromHexcodeToCodepoint } from 'emojibase';
+import cldrShortcodes from 'emojibase-data/en/shortcodes/cldr.json';
```

Comment:
https://emojibase.dev/docs/shortcodes#cldr

Reply:
`emojibase-data` is pretty big. 😕

Reply:
I thought we can fetch the required json files at build time and include them to the renovate package.

So we don't have the full data package as dependency ( using it as dev dependency instead )